webapp/cypress.config.ts

```diff
@@ -5,8 +5,7 @@ export default defineConfig({
   e2e: {
     baseUrl: "http://localhost:8080",
     apiUrl: "http://localhost:5001",
-    experimentalMemoryManagement: true,
-    numTestsKeptInMemory: 0,
+    numTestsKeptInMemory: 1,
```

Comment:
Actually the default is 50 here, so I'm just going to remove our config. It's very useful to have snapshots locally